common/src/main/java/org/corfudb/common/result/Result.java

```diff
@@ -126,13 +126,4 @@ public void ifError(Consumer<E> consumer) {
             return new Result<>(null, error);
         }
     }
-
-    public<U> Result<U, E> flatMap(Function<T, Result<U, E>> mapper) {
```

Comment:
this code is requred

Reply:
where is it being used?

From `Supplier.java`

```java

@FunctionalInterface
public interface Supplier<T> {

    /**
     * Gets a result.
     *
     * @return a result
     */
    T get();
} ```

Reply:
It's not used yet, but it's a part of `Result` API and should not be deleted

Reply:
ok

Reply:
Done.